scripts/filtered-temples.js

```diff
@@ -0,0 +1,100 @@
+function toggleMenu() {
+    document.querySelector('.nav-menu').classList.toggle('open');
+}
+
```

Comment:
You are missing the filter for each kind of temple and a different image for each temple. Please use this as an example:

https://github.com/HiddenSheave/wdd131/blob/week5/scripts/filtered-temples.js